setup.cfg

```diff
@@ -174,3 +174,9 @@ ignore_missing_imports = true
 
 [mypy-redis.sentinel]
 ignore_missing_imports = true
+
+[testenv:changelog]
+basepython = python3.8
```

Comment:
I think this would be better to not fix this to 3.8 since any Python3 that `django-redis` supports should work.

```suggestion
basepython = python3
```